<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_11">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="mainLayout" stretch="1,1,1">
      <item>
       <layout class="QVBoxLayout" name="left" stretch="1,1,8">
        <item>
         <widget class="QGroupBox" name="optionBox">
          <property name="title">
           <string>Option</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>端口：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="startService">
               <property name="text">
                <string>启动</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="stopService">
               <property name="text">
                <string>停止</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="testButton">
               <property name="text">
                <string>一键测试</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="statusBox">
          <property name="title">
           <string>Status</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLineEdit" name="statusEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="recognitonBox">
          <property name="title">
           <string>Recognition</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTabWidget" name="tabWidget">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab">
              <attribute name="title">
               <string>待识别</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout">
               <item row="1" column="0">
                <widget class="QPushButton" name="refreshUnrec">
                 <property name="text">
                  <string>刷新列表</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="recognizeAll">
                 <property name="text">
                  <string>一键识别</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="2">
                <widget class="QTableView" name="unrecTableView"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_2">
              <attribute name="title">
               <string>识别成功</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_8">
               <item row="0" column="0" colspan="2">
                <widget class="QTableView" name="successTableView"/>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="refreshSuccess">
                 <property name="text">
                  <string>刷新列表</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="submitSuccess">
                 <property name="text">
                  <string>一键提交</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_3">
              <attribute name="title">
               <string>识别失败</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="1" column="0">
                <widget class="QPushButton" name="refreshFail">
                 <property name="text">
                  <string>刷新列表</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="deleteFail">
                 <property name="text">
                  <string>一键删除</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="2">
                <widget class="QTableView" name="failTableView"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="middle" stretch="5,2,3">
        <item>
         <widget class="QGroupBox" name="captureBox">
          <property name="title">
           <string>Capture</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="sourceLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="plateBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Plate</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="resultLabel">
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="resultBox">
          <property name="title">
           <string>Result</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="plateColor">
             <property name="text">
              <string>车牌颜色</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="plateColorEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="plateStr">
             <property name="text">
              <string>车牌号码</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="plateStrEdit"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="reason">
             <property name="text">
              <string>违章原因</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="reasonEdit"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="location">
             <property name="text">
              <string>违章地点</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="locationEdit"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="right">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Record</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QLabel" name="carowner">
             <property name="text">
              <string>车主姓名</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="carownerEdit"/>
           </item>
           <item row="0" column="3">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>84</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="platenum">
             <property name="text">
              <string>车牌号码</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="platenumEdit"/>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="searchRecord">
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="idcard">
             <property name="text">
              <string>驾照号码</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLineEdit" name="idcardEdit"/>
           </item>
           <item row="2" column="3">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>84</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0" colspan="4">
            <widget class="QTableView" name="recordTableView"/>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QPushButton" name="selectAll">
             <property name="text">
              <string>全选</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2" colspan="2">
            <widget class="QPushButton" name="doDeduction">
             <property name="text">
              <string>一键扣款</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
